src/components/History/History.tsx

```diff
@@ -29,12 +30,25 @@ const History: React.FC<HistoryProps> = ({
       console.error('Failed to select chat')
       return
     }
-    const {messages} = await response.json()
+    const { messages } = await response.json()
     loadConversation(chat.id, messages.map((message: any) => {
-      return {
-        type: message.role,
-        content: message.content
-      } as Message
+      const baseMessage: BaseMessage = {
+        role: message.role,
+        content: message.content,
+      };
+
+      if (message.role === 'user') {
+        console.log(message)
```

Comment:
Flagging this log in case it was accidentally left in :)

Reply:
Oh yeah, thanks for flagging!